specs/daito/Multi_cloud_yig_integration.md

```diff
@@ -0,0 +1,325 @@
+# Multi-Cloud feature enhancements
+
+**Author(s)**: [Shufang Zeng](https://github.com/sfzeng) [Neelam Gupta](https://github.com/neelamgupta1491) [Ashit Kumar](https://github.com/kumarashit) 
+
+## Summary
+
+OpenSDS provides multi-cloud management platform. The Goal of multi-cloud is to enable data autonomy for multi-cloud environments. Currently OpenSDS multi-cloud (Gelato) provides multiple features for Object Data Migration, Lifecycle and Management across multiple clouds like AWS, Azure, GCP, Huawei, IBM. Also, Gelato need to increase the multi-cloud supportability matrix. In the next release, OpenSDS will add some key features, add new on-prem Object storage i.e. YIG and integrate with YIG to enhance the multi-cloud features of OpenSDS.
+The purpose of this design is to add new features and analyse and facilitate the OpenSDS multi-cloud (Gelato) integration with YIG (Yet Another Index Gateway)
+
+
+One of the Key components of this is 'S3 service'. S3 service handles s3 API requests for communicating with Object Storage backends. OpenSDS multi-cloud API gateway should be compatible with any s3 compatible API service. This s3 service should provide or should be flexible to provide all the multi-cloud features. [YIG](https://github.com/journeymidnight/yig ) is Object Storage Framework which is s3 compatible. YIG brings in multiple features which fits-in into the multi-cloud requirements. 
```

Comment:
nit: Key -> key  
"S3 service handles **s3** API..." -> ""S3 service handles **S3** API..."  
all occurrences of s3 -> S3 or vice versa consistently.  
"YIG is Object...." -> "YIG is an Object....."  
.."which fits-in into" -> "...which fits into the ..."